javascript/statements.json

```diff
@@ -615,46 +615,63 @@
             "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/default",
```

Comment:
I think this url is wrong.

Reply:
As explained here https://github.com/mdn/browser-compat-data/pull/585#issuecomment-341081341, we have three features here:
- import
- export
- default export

default exports are documented on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/default

Reply:
Makes sense.